packages/@aws-cdk/aws-appmesh/test/test.virtual-router.ts

```diff
@@ -216,6 +220,10 @@ export = {
           match: {
             prefixPath: '/',
           },
+          timeout: {
+            idle: cdk.Duration.seconds(9),
+            perRequest: cdk.Duration.seconds(9),
+          },
```

Comment:
No need for this (not used in an assertion).

Reply:
ack